### DISCUSSION TITLE
latitude longitude - ezdxf - DXF - set CRS for DXF file internally or via python - Stack Overflow

### DISCUSSION THREAD

**Discussion by iembry** [mozman/ezdxf#1008] [Q&A]
Hey everyone, can anyone answer the question that I posted some weeks ago? [https://stackoverflow.com/questions/77716058/ezdxf-dxf-set-crs-for-dxf-file-internally-or-via-python]:

The complete text is below:

I have a DXF file that was created via the Export Project to DXF function in QGIS 3.34. Although the exported DXF was set to EPSG 3395, that designation is not set in the dxf.coordinate_type.

Where is the XML information stored in the DXF file (refer to https://ezdxf.readthedocs.io/en/stable/dxfobjects/geodata.html) to set the CRS to EPSG 3395 Or can the CRS be defined via python?

This question is a pre-cursor to ezdxf -- transform undefined EPSG 3395 latitude longitude decimal degrees coordinates to WCS coordinates to save revised DXF file [https://stackoverflow.com/questions/77709215/ezdxf-transform-undefined-epsg-3395-latitude-longitude-decimal-degrees-coordi].

Thank you.

Irucka

**Comment by mozman** [comment]
Unfortunately I can't help you with that. I don't use the `GEODATA` feature myself and everything I know about it is in the documentation and possibly in the source code. I haven't touched this topic since implementing the `GEODATA` entity and the `geo` add-on.

**Comment by iembry** [comment]
OK. Manfred, thank you for the information.

I appreciate your assistance.

Irucka
